androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/triage/somatichealth/SomaticHealthScreen.kt

```diff
@@ -0,0 +1,193 @@
+                                ?: YesNo.UNKOWN,
+                            onSelection = {
+                                somaticHealthViewModel.isBloodInfection.value = it
+                            },
+                            choices = yesNoLabels.keys.toTypedArray(),
+                            labels = yesNoLabels
+                        )
+                        TitledTextField(
+                            title = "Typ",
+                            onChangeText = { somaticHealthViewModel.bloodInfectionType.value = it },
+                            textValue = somaticHealthViewModel.bloodInfectionType.value ?: ""
+                        )
+                    }
+                }
+            }
+            Row {
+                TitledSection(
+                    title = stringResource(id = R.string.multiresistant),
+                    modifier = Modifier
+                        .fillMaxWidth(0.5f)
+                        .padding(horizontal = 4.dp)
+                ) {
+                    //Multi-resistant content
+                    EnumRadioButtonsHorizontal(
+                        currentChoice = somaticHealthViewModel.isMultiresistant.value
+                            ?: YesNo.UNKOWN,
+                        onSelection = { somaticHealthViewModel.isMultiresistant.value = it },
+                        choices = yesNoLabels.keys.toTypedArray(),
+                        labels = yesNoLabels
+                    )
+                }
+
+                TitledSection(
+                    title = stringResource(id = R.string.suspicionGE),
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    // GE suspicion content
+                    EnumRadioButtonsHorizontal(
+                        currentChoice = somaticHealthViewModel.suspicionGE.value ?: YesNo.UNKOWN,
+                        onSelection = { somaticHealthViewModel.suspicionGE.value = it },
+                        choices = yesNoLabels.keys.toTypedArray(),
+                        labels = yesNoLabels
+                    )
+                }
+            }
+
+            TitledSection(title = stringResource(id = R.string.nursesNeeds)) {
+                Row {
+                    var isRendered by rememberSaveable { mutableStateOf(somaticHealthViewModel.isNursesNeed.value) }
+                    NursesNeeds(
+                        isNursesNeeds = somaticHealthViewModel.isNursesNeed.value ?: YesNo.UNKOWN,
+                        onNursesNeeds = {
+                            somaticHealthViewModel.isNursesNeed.value = it
+                            isRendered = somaticHealthViewModel.isNursesNeed.value
+                        },
+                        choices = yesNoLabels.keys.toTypedArray(),
+                        labels = yesNoLabels
+                    )
+                    if (isRendered?.equals(YesNo.YES) == true) {
```

Comment:
I would rather gray out components unless we're saving space by not showing items on a scrolling page. See Figma for reference

Reply:
Fixed it. Didn't use ```modifyIf()``` .
Didn't get it to work but included it in a TODO.